_pages/API/v1.9/DataService/026-dataservice-measurementsandvalues.md

```diff
@@ -0,0 +1,622 @@
+<h2 id="{{page.sections['dataservice']['secs']['measurementsAndValues'].anchor}}">{{page.sections['dataservice']['secs']['measurementsAndValues'].title}}</h2>
+
+<h3 id="{{page.sections['dataservice']['secs']['measurementsAndValues'].anchor}}-endpoints">Endpoints</h3>
+
+You can fetch, create, update and delete measurements and values using the following endpoints:
+<br/>
+
+>{{ site.images['info'] }} Endpoint `Measurements` creates measurements **without** measured values. Values sent to this endpoint are ignored and won't get saved! Always use the `Values` endpoint when creating or updating measurements **with** values!
+
+#### Measurements
+{% assign linkId="measurementsGetAll" %}
+{% assign method="GET" %}
+{% assign endpoint="/measurements" %}
+{% assign summary="Fetches measurements" %}
+{% capture description %}
+You can fetch all measurements or certain measurements only. Possible filter uri parameters are:
+
+{% capture table %}
+<code>Type</code> Parameter      |  Description <br> <code>Example</code>
+--------------------|-----------------------------------------------------------------------------------
+<nobr><code>Guid list</code> measurementUuids </nobr>         | Restricts the query to these measurements <br> `measurementUuids={5b59cac7-9ecd-403c-aa26-56dd25892421}`
+<nobr><code>Guid list</code> partUuids </nobr> | Restricts the query to these parts <br> `partUuids={e42c5327-6258-4c4c-b3e9-6d22c30938b2}`
+<nobr><code>Path</code> partPath </nobr> | Restricts the query to this part <br> `partPath=/metal%20part`
+<nobr><code>bool</code> deep </nobr><br><i>default:</i> <code>false</code> | Determines whether the query should affect all levels of the inspection plan. <br> `deep=true`
+<nobr><code>OrderCriteria</code> order </nobr><br><i>default:</i> <code>4 desc</code>   | Determines which attribute keys and which direction the keys should be ordered by <br> `order:4 asc, 10 desc`
+<nobr><code>Condition</code> searchCondition </nobr>| The query will only return items matching all conditions. A simple condition consists of the attribute key, followed by an operator and the value to match enclosed in brackets. Possible operators are: >, <, >=, <=, =, <>, In, NotIn, Like. <br> You can combine multiple conditions with '+'. It can be necessary to encode '+' as '%2B'. The format for date/time has to be “yyyy-mm-ddThh:mm:ssZ”. All values need to be surrounded by [ and ], multiple values are separated by a comma. <br> `searchCondition=4>[2012-11-13T00:00:00Z]`<br>`searchCondition=4>[2012-11-13T00:00:00Z]+852=[1]`<br>`searchCondition=10In[3,21,50]`
+<nobr><code>bool?</code> caseSensitive </nobr><br><i>default:</i> <code>null</code> | Determines weather the search condition string comparison should be done case sensitive, case insensitive or (if not selected) by the current database default. <br> `caseSensitive=true` <br> `caseSensitive=false`
```

Comment:
I think the concept of nullable bool is more suitable for the nuget, in the REST context a simple bool is enough. As default value I would suggest 'database default' instead of null, or we simply remove the default completely since it is explained on the right (since this is a special case were neither true/false is default).

Reply:
Yes, this makes sense. I have made it non-nullable - thanks for the feedback.